eq-author/src/apollo/createApolloErrorLink.js

```diff
@@ -2,8 +2,8 @@ import { onError } from "apollo-link-error";
 import { apiDownError } from "../redux/saving/actions";
 
 export default getStore =>
-  onError(({ networkError }) => {
-    if (networkError) {
+  onError(({ networkError, operation: { operationName } }) => {
+    if (networkError && operationName !== "GetCurrentUser") {
```

Comment:
If we don't send the request if we have no token then we shouldn't need this

Reply:
The reason for sending the request is because of the network only fetch policy. This was to avoid displaying the users avatar even after they have logged off (since Apollo caches the user locally in its store). I think this could be avoided by using the presence of an Auth token to determine if the user avatar should be displayed, I'll take a look tomorrow.